future_challenges_for_RSEng.tex

```diff
@@ -29,11 +29,15 @@
 Nick del Grosso\texorpdfstring{\autref{1}}{}
 } % Authors and references to addresses
 \institute{\autlabel{1} Fantasy University} % Institutes with labels
-\abstract{Admittedly, it’s still only 2025 and we are actually stillworking on establishing Research Software Engineering
-as a new domain with the hope of ensuring that it gets the recognition it deserves.
-And while it is certainly too early to already herald the end of RSEng,
-we should acknowledge that the future is in constant flux and
-it is worthwhile to discuss how RSEng will/could change in response to these upcoming challenges in an open discussion.} % Abstract of the article
+\abstract{It's 2025 and we are, admittedly, still working on establishing and growing Research Software Engineering (RSE)
+as a domain in its own right, with the hope of ensuring that it gets the recognition it deserves.
+Nonetheless, as research becomes ever more computational in nature, there are already people suggesting that RSE will
```

Comment:
I don't think it necessarily does. In some fields, it manifests as more data-processing (which technically is computational work as well, but then what isn't?).

Reply:
I'll merge and rephrase it with: as the use of digital tools becomes ever more pervasive in a researchers day-to-day tasks,